base-template.sh

```diff
@@ -421,19 +439,52 @@ update_shared_hooks_if_appropriate() {
 #   1 in case a hook fails, 0 otherwise
 #####################################################
 execute_shared_hooks() {
-    for SHARED_ROOT in ~/.githooks/shared/*; do
-        REMOTE_URL=$(cd "$SHARED_ROOT" && git config --get remote.origin.url)
+    # split on comma and newline
+    IFS=",
+    "
+    for SHARED_REPO in $SHARED_REPOS_LIST; do
+
+        set_shared_root "$SHARED_REPO"
+
+        # Fail if the shared root is not available (if enabled)
+        FAIL_ON_NOT_EXISTING=$(git config --global --get githooks.failOnNotExistingSharedHooks)
+        
+        if [ "$FAIL_ON_NOT_EXISTING" = "true"]; then
+            if [ ! -d "$SHARED_ROOT" ] || [ ! -d "$SHARED_ROOT/.git" ]; then
```

Comment:
`if [ ! -d "$SHARED_ROOT/.git" ]; then` should be enough because if the `.git` directory is not there, it's parent won't be either.